.github/ISSUE_TEMPLATE/Inbound_Review.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Inbound Review
+about: Ticket for requesting an Inbound Review
+title: Request an Inbound Review
+labels: helpdesk
+assignees: ''
+
+---
+# Inbound Review
+
```

Comment:
I would add a description with more details about the Inbound Review checklist so users have all the information they need -- you can steal the text in the checklist itself for this. We should add a link to the checklist as well: https://github.com/DSACMS/ospo-guide/blob/main/inbound/inbound-review-checklist.md